hadoop-tools/hadoop-azure/dev-support/testrun-scripts/runtests.sh

```diff
@@ -248,10 +263,11 @@ do
          ;;
       *) logOutput "ERROR: Invalid selection"
       ;;
-   esac
-done
+    esac
+  done
+fi
 
-if [ $runTest == true ]
+if [[ $runTest == true && $"$IS_CRON_JOB" != "true" ]]
```

Comment:
Can this be simplified??
Not sure if bash allows something like this" `$runTests && !$IS_CRON_JOB`

Reply:
Changed to `$runTest && $"$IS_CRON_JOB" != "true"`
Env variables are only allowed as string in cronjob hence the IS_CRON_JOB part